docs/contributing.rst

```diff
@@ -179,29 +179,31 @@ Scrapy:
   See https://help.github.com/en/github/using-git/setting-your-username-in-git for
   setup instructions.
 
+.. _scrapy-pre-commit:
+
 Pre-commit
 ==========
 
-We use `pre-commit`_ to automatically address simple code issues before every 
+We use `pre-commit`_ to automatically address simple code issues before every
 commit.
 
 .. _pre-commit: https://pre-commit.com/
 
-Before you start writing a patch:
+After your create a local clone of your fork of the Scrapy repository:
```

Comment:
I wrote the original phrasing, but I felt it was misleading, i.e. you do not need to do this before every patch, but after every clone.